src/network/clientpackethandler.cpp

```diff
@@ -1343,15 +1344,18 @@ void Client::handleCommand_HudSetSky(NetworkPacket* pkt)
 	*pkt >> skybox.bgcolor >> skybox.type >> skybox.clouds >>
 		skybox.fog_sun_tint >> skybox.fog_moon_tint >> skybox.fog_tint_type;
 
-	if (skybox.type == "skybox") {
+	if (skybox.type == "skybox" ||
```

Comment:
This check appears quite a few times. Why not make it a helper function in the `SkyboxParams` struct? e.g.

```c++
bool hasTextures() const
{
    return skybox.type == "skybox" || skybox.type == "skybox_back" || skybox.type == "skybox_front";
}
```

Reply:
This is a tricky one as different checks for different skybox types need to be preformed in various parts of the code. If I add checker functions to simplify it, I'd rather they be universal, instead of a local boolean that's only going to be used two or three times in each file. I'll try adding a few of those checks in `skybox.h` for commonly used combinations and see if they work everywhere.